bin/setupSecurityserviceTest.sh

```diff
@@ -1,15 +1,12 @@
 #!/bin/bash
 
-COMPOSE=$(dirname "$0")/../$(ls $(dirname "$0")/../ | awk '/docker-compose/ && !/test-tools/')
-OT=$(docker-compose -f ${COMPOSE} run --rm --entrypoint /edgex/security-proxy-setup edgex-proxy --init=false --useradd=jerry --group=admin | grep '^the access token for')
-export TOKEN=$( echo $OT | sed 's/.*: \([^.]*\.[^.]*\.[^.]*\).*/\1/')
-
-#echo TOKEN=$TOKEN
+export TOKEN=`./bin/security/setupSecurityAccount.sh -useradd`
```

Comment:
The output of the command includes  `TOKEN=Info: Add security account.` with the actual token, so it causes the securityTest error.